app/Http/Controllers/DiagnosticsController.php

```diff
@@ -189,9 +190,13 @@ public function get_errors()
 			}
 		}
 
-		$version_num = implode('.', array_map('intval', str_split($settings['version'], 2)));
-		if ($this->lycheeVersion->isRelease && $version_num < $this->versions['Lychee']) {
-			$errors[] = 'Error: Database is behind file versions. Please apply the migration.';
+		if (isset($settings['version'])) {
+			$version_num = implode('.', array_map('intval', str_split($settings['version'], 2)));
+			if ($this->lycheeVersion->isRelease && $version_num < $this->versions['Lychee']) {
+				$errors[] = 'Error: Database is behind file versions. Please apply the migration.';
```

Comment:
Existing wording, but I wonder if it's worth mentioning `php artisan migrate` here?

I've also wondered whether it would be worth having the migration part of the updater available in release versions...

Reply:
I was also considering adding something to apply migrations from the web.
Especially for those with only FTP access.